spikeextractors/extractors/bindatrecordingextractor/bindatrecordingextractor.py

```diff
@@ -77,7 +77,7 @@ def get_traces(self, channel_ids=None, start_frame=None, end_frame=None):
             exp_idx = self._dtype.find('int') + 3
             exp = int(self._dtype[exp_idx:])
             recordings = recordings.astype('float32') - 2**(exp - 1)
-        if self._gain is not None:
+        if self._gain is not None and return_scaled:
             recordings = recordings * self._gain
```

Comment:
I gain is None should and return_scaled=True should raise an error

Reply:
I think we should do the following. If gain is None in the constructor, we set it to one. Then we multiply by whatever gain we have. I use the bindat a lot (cache) to save filtered data already converted to uV. In that case the scale should be one, bit I think it should be so by default.

Reply:
Fixing BinDat on another PR